LOGGING/test/Log4NetAppender.Tests/Log4NetAppender.Tests.csproj

```diff
@@ -2,9 +2,10 @@
 
   <PropertyGroup>
     <RootNamespace>Microsoft.ApplicationInsights.Log4NetAppender.Tests</RootNamespace>
-    <AssemblyName>Microsoft.ApplicationInsights.Log4NetAppender.Net45.Tests</AssemblyName>
-    <TargetFramework>net45</TargetFramework>
+    <AssemblyName>Microsoft.ApplicationInsights.Log4NetAppender.Tests</AssemblyName>
+    <TargetFrameworks>net45;netcoreapp1.1;netcoreapp2.1;netcoreapp3.1</TargetFrameworks>
```

Comment:
we can remove netcoreapp1.1 target in this PR itself. Or we can do all the "removal of 1.1"
 separately.

Reply:
yeah, first we will just add all and we have a separate issue to remove 1.1 :) thanks for reviewing!